features/importing/archivist.feature

```diff
@@ -49,80 +49,82 @@ Feature: Archivist bulk imports
    
   Scenario: Importing only sends one email even if there are many works
   
-    Given I have an archivist "elynross"
+    Given I have an archivist "archivist"
       And the default ratings exist
-    When I am logged in as "elynross"
+    When I am logged in as "archivist"
       And I import the works "http://ao3testing.dreamwidth.org/593.html, http://ao3testing.dreamwidth.org/325.html"
       And the system processes jobs
     Then 1 email should be delivered to "ao3testing@dreamwidth.org"
 
-  Scenario: Importing sends a different email if you're already an author on the archive
+  Scenario: Importing for an existing Archive author should have correct byline and email
 
   Given the following activated user exists
     | login | email                     |
     | ao3   | ao3testing@dreamwidth.org |
-    And I have an archivist "heathercook"
+    And I have an archivist "archivist"
     And the default ratings exist
-  When I am logged in as "heathercook"
+  When I am logged in as "archivist"
     And I import the work "http://ao3testing.dreamwidth.org/593.html"
   Then I should see import confirmation
+    And I should see "ao3"
+    And I should not see "[archived by archivist]"
```

Comment:
FYI, james, it is testing that in the I should see "ao3" line (that's the name of the user)